<launch>
  <node pkg="video_publisher" name="video_publisher_node" type="video_publisher_node" output="screen">
  <param name="show_video_flag" type="bool" value="true"/>
  <param name="published_video_file_name" type="string" value="/home/ubuntu/ros_my_workspace/src/video_publisher/video/a544.avi"/>
  <param name="given_video_rate" type="double" value="30.0"/>
  </node>
</launch>